### PR TITLE
Update ediscovery-troubleshooting-common-issues.md

### PR DESCRIPTION
**Pull request by Sokarpi**
We have some new error messages that are not yet documented . I added two new messages and their explanations here.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/ediscovery-troubleshooting-common-issues.md [hunk 1]
```diff
@@ -86,6 +86,20 @@ When running an eDiscovery search that includes SharePoint Online and One Drive
 
 2. Use the procedures at [Manually request crawling and re-indexing of a site, a library, or a list](/sharepoint/crawl-site-content) to reindex the site.
 
+## Error/issue: This file wasn't exported because it doesn't exist anymore. The file was included in the count of estimated search results because it's still listed in the index. The file will eventually be removed from the index, and won't cause an error in the future.
+
+You may see that error when running an eDiscovery search that includes SharePoint Online and One Drive For Business locations. eDiscovery relies on teh SPO index to identify the file locations. If the file was deleted but the SPO index was not yet updated this error may occur.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery-troubleshooting-common-issues.md:91]:
```suggestion
You may see this error when running an eDiscovery search that includes SharePoint Online and One Drive for Business locations. eDiscovery relies on the SPO index to identify the file locations. If the file was deleted but the SPO index was not yet updated this error may occur.
```

File: microsoft-365/compliance/ediscovery-troubleshooting-common-issues.md [hunk 2]
```diff
@@ -86,6 +86,20 @@ When running an eDiscovery search that includes SharePoint Online and One Drive
...
+
+### Resolution 
+Open the SPO location and verify that this file indeed is not there.
+Suggested solution is to manually reindex the site, or wait till the site reindexes by the automatic background process.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery-troubleshooting-common-issues.md:95]:
```suggestion
The suggested solution is to manually reindex the site, or wait until the site reindexes by the automatic background process.
```

File: microsoft-365/compliance/ediscovery-troubleshooting-common-issues.md [hunk 3]
```diff
@@ -86,6 +86,20 @@ When running an eDiscovery search that includes SharePoint Online and One Drive
...
+
+
+## Error/issue: This search result was not downloaded as it is a folder or other artefact that can't be downloaded by itself, any items inside the folder or library will be downloaded.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery-troubleshooting-common-issues.md:98]:
```suggestion
## Error/issue: This search result was not downloaded as it is a folder or other artifact that can't be downloaded by itself; any items inside the folder or library will be downloaded.
```

File: microsoft-365/compliance/ediscovery-troubleshooting-common-issues.md [hunk 4]
```diff
@@ -86,6 +86,20 @@ When running an eDiscovery search that includes SharePoint Online and One Drive
...
+
+You may see that error when running an eDiscovery search that includes SharePoint Online and One Drive For Business locations. It means that we were going to try and export the item reported in the index, but it turned out to be a folder so we did not export it. As mentioned in the error, we don't export folder items but we do export their contents.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery-troubleshooting-common-issues.md:100]:
```suggestion
You may see this error when running an eDiscovery search that includes SharePoint Online and One Drive for Business locations. It means that we tried to export the item reported in the index, but it turned out to be a folder so it was not exported. As mentioned in the error, we don't export folder items but we do export their contents.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Sokarpi, please commit my suggested changes if you agree with them. Many thanks.

**Comment by markjjo** [comment]
@Sokarpi PR merged. Thanks for submitting these new errors.

**Comment by Sokarpi** [comment]
Thank you Mark!

Sonya Karpishpan
Support Escalation Engineer | Office 365 Security & Compliance | Equivio
***@***.******@***.***>| +1 301 771 8687 | Manager: Ryan Jones ***@***.******@***.***>
[MSFT_logo_Gray DE sized SIG1.png]

Join our Preview Programs<http://aka.ms/mip-preview>
Resources: Documentation<https://aka.ms/MIPCDocs>, MIP&C Blog<https://aka.ms/MIPblog>, Tech Community<https://aka.ms/ComplianceTechCommunity>, Yammer<https://aka.ms/askMIPteam>
MIP product feedback via UserVoice<https://aka.ms/MIPCUserVoice> | Compliance product feedback via UserVoice<http://aka.ms/CompUV>
Check out the sample files and documents on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnihendle%2FMIP-Comp%2Ftree%2Fmaster%2FMIP%2FWebinars%2FUnified%2520Labeling%2520Migration&data=02%7C01%7CSonya.Karpishpan%40microsoft.com%7C5bd37595404b4df614f308d7bc5d1658%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637184980306867818&sdata=9GX5LmJvnFb8%2BJe9Sh%2Bn5CrqDUrl3uSQymkZg2FEe9U%3D&reserved=0>

Follow us on Twitter @MipNews

**Pull request merged**